src/pixie.c

```diff
@@ -35,3 +41,36 @@ void pixie_attack(void) {
 	PIXIE_FREE(ehash1);
 	PIXIE_FREE(ehash2);
 }
+
+int pixie_wrapper(char *cmd) {
+	struct stat dirstat;
+	FILE *fp = NULL;
+	char file[FILENAME_MAX];
+	char *sh = NULL;
+	int ret_val = EXIT_FAILURE;
+
+	if (stat(CONF_DIR, &dirstat) == 0) {
+		/* Check directory used to store *.wpc files */
+		snprintf(file, FILENAME_MAX, "%s/pixiewps-wrapper.sh", CONF_DIR);
```

Comment:
i'd vote for simply calling "pixie-wrapper" or "pixiewps-wrapper" without any file extension and any hardcoded path. so the user can decide which scripting language he wants to use and where to put the file, and then simply make it available via `$PATH` env var.